src/main/frontend/format/mldoc.cljs

```diff
@@ -115,10 +115,10 @@
   (if (seq ast)
     (let [original-ast ast
           ast (map first ast)           ; without position meta
-          directive?
-          (fn [[item _]] (= "directive" (string/lower-case (first item))))
+          directive? (fn [[item _]] (= "directive" (string/lower-case (first item))))
           grouped-ast (group-by directive? original-ast)
-          directive-ast (get grouped-ast true)
+          directive-ast (first (partition-by directive? original-ast))
+          directive-ast (when (every? directive? directive-ast) directive-ast)
```

Comment:
I think this line is redundant.
The above line `partition-by` already ensures every item in the directive-ast is a directive.

Reply:
This check is for document without #+keywords in pre-block. Its `(first (partition-by directive? original-ast))` would be the ast of block, which leads to error later.

Reply:
gotcha, how about `(take-while directive? original-ast)` here?